### DISCUSSION TITLE
How to Search Products Using Custom Fields from a Product Extension Module in MedusaJS 2.x ?

### DISCUSSION THREAD

**Discussion by soufianetomase** [medusajs/medusa#9908] [Q&A]
I have created a new module called product_extension that introduces additional fields for products. I have successfully linked this module to the product entity. Now, I am trying to search for products using api based on a specific field within the product_extension module. How can I achieve this?

**Comment by SalahAdDin** [comment]
Did you check the Mailisearch or Typesense modules?
